src/components/FeaturePanel/FeaturePanel.tsx

```diff
@@ -95,7 +94,9 @@ export const FeaturePanel = ({ headingRef }: FeaturePanelProps) => {
 
               <PanelSidePadding>
                 {!movePropertiesBelowMembers && <PropertiesComponent />}
-                <RouteDistributionInPanel feature={feature} />
+                {isClimbingCrag(feature) && (
+                  <RouteDistribution features={feature.memberFeatures} />
+                )}
```

Comment:
Can you please reintroduce `<RouteDistributionInPanel/>` without any props as it was even before you added `feature={feature}`? 

FeaturePanel code has to stay uncomplicated, thats why there has to be special component which will handle the condtions. It is written in the note on line 55. 🙂

Reply:
Same with `<ClimbingTypeBadge feature={feature} />` 🙏